contracts/savings/SavingsManager.sol

```diff
@@ -118,6 +130,34 @@ contract SavingsManager is ISavingsManager, PausableModule {
         IERC20(_mAsset).safeApprove(address(_savingsContract), uint256(-1));
     }
 
+    /**
+     * @dev Freezes streaming of mAssets
+     */
+    function freezeStreams()
+        external
+        onlyGovernor
+        whenStreamsNotFrozen
+    {
+        streamsFrozen = true;
```

Comment:
Should it be possible to also "unfreeze" a stream?

Reply:
The freezing was just for doing a migration from one SavingsManager to the next.. however now I will add in a 'whenNotPaused' instead.

Reply:
I see. Thanks!